soc/nxp/imx/imx8m/Kconfig.soc

```diff
@@ -40,7 +40,7 @@ config SOC_MIMX8ML8_ADSP
 	help
 	  Enable support for NXP i.MX 8MPLUS Audio DSP
 
-config SOC_MIMX8MP_M7
+config SOC_MIMX8ML8_M7
```

Comment:
Squash all board commits to one commit then a separate commit for both manifest updates so that chance of breaking git bisect is reduced

Reply:
done.